types/address/store_key_test.go

```diff
@@ -3,12 +3,19 @@ package address_test
 import (
 	"testing"
 
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/cosmos/cosmos-sdk/types/address"
 )
 
-func TestLengthPrefixedAddressStoreKey(t *testing.T) {
+func TestStoreKeySuite(t *testing.T) {
+	suite.Run(t, new(StoreKeySuite))
+}
+
+type StoreKeySuite struct{ suite.Suite }
+
+func (suite *StoreKeySuite) TestLengthPrefix() {
```

Comment:
I converted this test to Test Suite. Using test suites is superior because we have better options on managing which tests to run and have better functionality to setup and tear down. So by default let's  use it everywhere.